### DISCUSSION TITLE
only Arm64 Docker images on Docker hub for 2.0 version of Duende

### DISCUSSION THREAD

**Discussion by markogl** [skoruba/Duende.IdentityServer.Admin#208] [Q&A] [answered]
Hi,

I am new to Docker images and as I have problems with managing to build images using docker compose build from source I wanted to try using images directly from Hub. Well, there are only arm64 versions and I do not know how to run them on Windows PC (amd 64). Can anyone help?

Thanx.

**Comment by skoruba** [comment, marked as answer]
I will push docker image also for windows. 

Thanks!
